cdc/processor/processor.go

```diff
@@ -1007,10 +1007,6 @@ func (d *ddlHandler) Run(ctx context.Context, _ ...chan<- error) error {
 			if jobEntry.OpType == model.OpTypeResolved {
 				d.schemaStorage.AdvanceResolvedTs(jobEntry.CRTs)
 			}
-			err := jobEntry.Err
```

Comment:
why remove this?

Reply:
I think this error won't be checked, because it's already checked when the error happens.